src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -102,6 +102,17 @@ internal static void OnDotNetRuntimeLoaded(this TraceProcess process, TraceLoade
             if (map[process.ProcessIndex].OnLoaded != null) map[process.ProcessIndex].OnLoaded(runtime);
         }
 
+        internal static void OnDotNetRuntimeRemoved(this TraceProcess process)
```

Comment:
Lets stick with the 'Start-Stop' terminology for things.  Below I suggests OnProcessRemoved be renamed OnProcessStop, in the same way I think we should name this OnDotNetRuntimeStop.

Reply:
Done.  The interesting thing there is that there is a public AddCallbackOnDotNetRuntimeLoad, and the Start/Stop methods are internal.  If this were exposed externally perhaps we would use Unload?  Though I feel it is not necessary since ProcessStop has the same semantic as RuntimeStop, where as ProcessStart and RuntimeLoad happen on different events.

Reply:
Unload has been used in some places (dlls are loaded and unload not started and stopped).   I see that we use 'Loaded' for the runtime start event, so I am OK using Unload for the stop.

Reply:
Done.